content/en/docs/concepts/configuration/poseidon-firmament-alternate-scheduler.md

```diff
@@ -0,0 +1,113 @@
+
+
+{{< note >}}
+For details about the design of this project see the [design document](https://github.com/kubernetes-sigs/poseidon/blob/master/docs/design/README.md).
+{{< /note >}}
+
+## Possible Use Case Scenarios - When to use it  
+
+As mentioned earlier, Poseidon/Firmament scheduler enables extremely high throughput scheduling environment at scale due to its bulk scheduling approach superiority versus K8S pod-at-a-time approach. In our extensive tests, we have observed substantial throughput benefits as long as resource requirements (CPU/Memory) for incoming Pods are uniform across jobs (Replicasets/Deployments/Jobs), mainly due to efficient amortization of work across jobs.
+
+Although, Poseidon/Firmament scheduler is capable of scheduling various types of workloads (service, batch, etc.), following are the few use cases where it excels the most:  
+1. For “Big Data/AI” jobs consisting of large number of tasks, throughput benefits are tremendous.  
+2. Substantial throughput benefits also for service or batch job scenarios where workload resource requirements are uniform across jobs (Replicasets/Deployments/Jobs).  
+
+## Current Project Stage	
+
+- **Alpha Release - Incubation repo.** at https://github.com/kubernetes-sigs/poseidon.  
+- Currently, Poseidon-Firmament scheduler **does not provide support for high availability**, our implementation assumes that the scheduler cannot fail. The [design document](https://github.com/kubernetes-sigs/poseidon/blob/master/docs/design/README.md) describes possible ways to enable high availability, but we leave this to future work.  
+- We are **not aware of any production deployment** of Poseidon-Firmament scheduler at this time.  
+
+## Features Comparison Matrix	
+
+
+|Feature|Kubernetes Default Scheduler|Poseidon/Firmament Scheduler|Notes|
+|--- |--- |--- |--- |
+|Node Affinity/Anti-Affinity|Y|Y||
+|Pod Affinity/Anti-Affinity - including support for pod anti-affinity symmetry|Y|Y|Currently, throughput numbers are definitely better for default scheduler versus Poseidon/Firmament due to the recent pod affinity/anti-affinity optimizations within K8S (release 1.11 & 1.12). Poseidon/Firmament scheduler currently “drip-feeds” pods into the scheduling algorithm and processing is one-pod-at-a-time, exactly similar to how default scheduler does. We initially tried achieving pod affinity/anti-affinity functionality using native flow network constructs (convex arc costs, “XOR” & “AND” flow network constructs as described in Ionel Gog’s PHD thesis) but we ran into few challenges and could not make it work.Using the flow network constructs we intend on achieving similar throughput benefits for affinity/anti-affinity pods as we are currently seeing for normal pods.|
```

Comment:
Similar comment as above. This paragraph has too much details about the reasons for lower performance of Poseidon. These are not useful in a user doc. I think you should only point out that the default scheduler is faster in such scenarios and your plans to address it in the future.

Reply:
@bsalamat This is fixed.